MusicNotationKit/MusicNotationKitTests/MeasureTests.swift

```diff
@@ -21,7 +21,9 @@ class MeasureTests: XCTestCase {
 			timeSignature: TimeSignature(topNumber: 4, bottomNumber: 4, tempo: 120),
 			key: Key(noteLetter: .C))
 		XCTAssertEqual(measure.notes.count, 0)
-		measure.addNote(Note(noteDuration: .Whole))
-		XCTAssertEqual(measure.notes.count, 1)
+		measure.addNote(Note(noteDuration: .Whole, tone: Tone(noteLetter: .C, octave: .Octave0)))
+		measure.addNote(Note(noteDuration: .Quarter, tone: Tone(noteLetter: .D, octave: .Octave0)))
+		XCTAssertEqual(measure.notes.count, 2)
+		debugPrint(measure)
```

Comment:
You do not need to call `debugPrint`. I think you should be able to just call `print`. If it doesn't work, let me know. Because that is what I want it to do. I didn't even know there was a `debugPrint` method.